src/renderer/components/_molecules/project-tree/index.tsx

```
@@ -253,30 +380,72 @@ const ProjectTreeLeaf = ({ leafLang, label, ...res }: IProjectTreeLeafProps) =>
-        </button>
+          onBlur={(e) => handleRename(e.target.value || '')}
+          className='w-full px-1 text-xs border-0 bg-transparent text-neutral-850 dark:text-neutral-300 focus:outline-none'
+        />
+      ) : (
+        <span
+          className={cn(
+            'ml-1 w-[90%] overflow-hidden text-ellipsis whitespace-nowrap font-caption text-xs font-normal text-neutral-850 dark:text-neutral-300',
+            name === label && 'font-medium text-neutral-1000 dark:text-white',
+          )}
+          onDoubleClick={() => setEditing(true)}
+          dangerouslySetInnerHTML={{ __html: label || '' }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)